app/Models/Feed.php

```diff
@@ -23,6 +23,7 @@ class FreshRSS_Feed extends Minz_Model {
 	private $priority = self::PRIORITY_MAIN_STREAM;
 	private $pathEntries = '';
 	private $httpAuth = '';
+        private $tolerateInvalidSSLCertificate = false;
```

Comment:
Btw, there are some indentation errors like these (spaces instead of tabs). See the CI report for specifics but they also jump out here on GH. :-)

Reply:
Thanks. I should have mentioned it because I noticed this when commiting my changes. I wanted to submit my current state "as is" to make sure the direction was correct.
Good to see the test suite is watching this kind of mistakes 😉